qcodes/instrument_drivers/QuTech/S5i.py

```diff
@@ -14,43 +14,49 @@ class S5i(Instrument):
 
     Args:
         name (str): name of the instrument.
-
         spi_rack (SPI_rack): instance of the SPI_rack class as defined in
             the spirack package. This class manages communication with the
             individual modules.
-
         module (int): module number as set on the hardware.
     """
 
     def __init__(self, name, spi_rack, module, **kwargs):
         super().__init__(name, **kwargs)
 
-        self.s5i = S5i_module(spi_rack, module)
+        self.s5i = S5i_module(spi_rack, module, frequency=41e6, enable_output=1,
+                              output_level=0)
 
-        self.add_parameter('use_external_reference',
-                           label='Use external reference',
-                           get_cmd=self._use_external_reference,
-                           set_cmd=self.s5i.use_external_reference,
-                           vals=Bool())
+        self.add_parameter('enabled',
```

Comment:
Isn't this a better name? because `S5i.enabled(False)` looks like the whole instrument is disabled, while what's happening is `S5i.output_enabled(False)`.
```suggestion
        self.add_parameter('output_enabled',
```

Reply:
done